Source/bmalloc/libpas/src/libpas/jit_heap.c

```diff
@@ -54,12 +54,15 @@ void jit_heap_add_fresh_memory(pas_range range)
 {
     static const bool verbose = false;
 
-    if (verbose)
-        pas_log("JIT heap at %p...%p\n", (void*)range.begin, (void*)range.end);
+    if (verbose || PAS_SHOULD_LOG(PAS_LOG_JIT_HEAPS))
```

Comment:
Why not just change the definition of `verbose` to be:
```
static const bool verbose = false || PAS_SHOULD_LOG(PAS_LOG_JIT_HEAPS);
```

Then, you don't have to make is many edits, and the code is also more compact and easier to read.

Reply:
Good point, will change